styles/globals.css

```diff
@@ -34,7 +34,7 @@ li {
 button {
   font-size: 20px;
   color: #ffffff;
-  background-color: #009eff;
+  background-color: #1917a1;
```

Comment:
@niteshseram Can we please get a better color for the primary button? The changed one is a bit darker. I understand, it was for the contrast ratio fix but see of you can suggest a better color from the blue palette?

Reply:
Can we use the same color as this?
![Capture](https://user-images.githubusercontent.com/51370246/135276925-500ce457-c892-4824-beb4-a8bc8ad97342.PNG)

Reply:
Or are you suggesting using `#1917a1` instead?

Reply:
Does this work? `#24a0ed`

Reply:
Another option is `#065b8e`

Reply:
I think we should better fix a blue palette from tailwind colors. It will make things easier when we are writing the CSS in the tailwind.

Reply:
Sure, let's go ahead with that if it satisfies the contrast ratio as well.. It is anyway subject to change later.

Reply:
Since we are using blue.700 hex code, we can still later use this color when writing CSS using tailwind